### ISSUE TITLE
Feature request: Honor "Edit Image" Preview Focus selection

### ISSUE THREAD

**Issue by LorenAmelang** [Dimillian/IceCubesApp#1090]
When I post images from the Mastodon web client, the Edit button leads to an Edit Image view with ALT text entry, and the ability to select which point on the image should be shown in all client views. When the post is viewed in Mastodon web, the selected critical difference between left and right image, at the bottom right, is shown: 

![Mastodon web shows chosen bottom icons](https://user-images.githubusercontent.com/1297090/221441623-7c547679-08f3-494c-b373-55fc59060bb9.png)

On my iPhone with maximum image size selected, the whole image shows, including the very bottom: 

![iPhone-2023 02 25-21 29 14 000 iPhone large shows all](https://user-images.githubusercontent.com/1297090/221441702-752d6098-646c-44a5-8317-174ef73623be.png)

The iPhone with smaller image selection shows the middle of the images, missing the important point: 

![iPhone-2023 02 25-21 30 46 000 iPhone small ignores focus](https://user-images.githubusercontent.com/1297090/221441823-f6373d42-955a-4d8c-831e-bbfda2b1d410.png)

My iPad, with space to spare, shows an even smaller middle view, even with maximum image size chosen - ignoring the selected focus point: 

![iPad-2023 02 25-21 21 28 000 iPad large media - default center view](https://user-images.githubusercontent.com/1297090/221441905-0676b6ed-1065-410a-8d43-e2a2dad0a69b.png)

Could we at least have the full image option on iPad? And hopefully someday proper handling of the chosen image focus?
